custom message when table empty

axcooper opened the pull request:
## Pull request overview
Adding custom message where NPS page is empty

## Pull request checklist

Please check if your PR fulfills the following:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been run locally and are passing (`shinytest2::test_app()`)
- [x] Code is styled according to tidyverse styling (checked locally with `styler::style_dir()` and `lintr::lint_dir()`)

## What is the current behaviour?
Generic message of no rows found.



## What is the new behaviour?

![image](https://github.com/user-attachments/assets/338c62bf-3535-453d-a7f8-289b7568abf4)
...

## Anything else

axcooper committed:
custom message when table empty

johalastrahol reviewed (approved):
Does what says on the tin - message: no rows found

rmbielby committed:
Merge branch 'main' into 52-page-5-add-custom-message-when-no-rows-are-found-in-the-table

axcooper commented:
message: no rows found is the default - have changed to 'No results for this provider in this year'

johalastrahol committed:
sorting merge conflict for manifest checksum

johalastrahol committed:
pull push to sort manifest chcksum

Merge branch '52-page-5-add-custom-message-when-no-rows-are-found-in-the-table' of https://github.com/dfe-analytical-services/apprenticeships-provider-dashboard into 52-page-5-add-custom-message-when-no-rows-are-found-in-the-table

# Conflicts:
#	manifest.json

axcooper commented:
Not currently letting me approve the changes
